### PR TITLE
adding draft on Chainguard Enforce profiles

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

New guide on how to use profiles in Chainguard Enforce.

### What should this PR do?

Resolves https://github.com/chainguard-dev/internal/issues/1478

### How should this PR be tested?

Reviewed for copy as well as technical accuracy.

### CODE REVIEW

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 90
+toc: true
+---
+
+When you install Chainguard Enforce, you have the option to specify a *profile* that will determine what permissions Chainguard Enforce has over your cluster. Currently, Enforce offers two specialized profiles: `enforcer` and `observer`.
+
+The `observer` profile has less expansive permissions, making it useful for running Enforce in a sort of "read-only" mode. The `enforcer` profile, on the other hand, has broader permissions that enable you to benefit from all of Chainguard Enforce's features, including policy distribution and enforcement.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md:19]: I'm not sure what "sort of 'read-only'" means here, but is there more precise language that can be used, or can you say it is "read-only" or "virtually read-only"?

Also subject-verb agreement here: "enforce profile ... enables you..."

```suggestion
The `observer` profile has less expansive permissions, making it useful for running Enforce in a sort of "read-only" mode. The `enforcer` profile, on the other hand, has broader permissions that enables you to benefit from all of Chainguard Enforce's features, including policy distribution and enforcement.
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+## Observer Profile
+
+To enroll a cluster into Chainguard Enforce under the `observer` profile, include the `--profiles=` option followed by `observer`, as in the following example.
+
+```sh
+chainctl cluster install --profiles=observer
+```
+
+As its name suggests, the `observer` profile is essentially a "readonly" profile for Chainguard Enforce. When installed under the `observer` profile, Chainguard Enforce does not install an admission webhook onto the cluster. This means that Enforce will only be able to monitor the cluster and report violations; it won't be able to enforce policies or make any changes to your clusters.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md:32]: I would be consistent with "read-only" or "readonly" or "read only" throughout

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+## Enforcer Profile
+
+`enforcer` is the default profile; if you install Chainguard Enforce without specifying a profile, the cluster will automatically be enrolled into the `enforcer` profile. 
+
+```sh
+chainctl cluster install
+```
+
+You can, however, specify that you want to install the the `enforcer` profile by including the `--profiles=` option and passing it the `enforcer` value
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md:47]:
```suggestion
You can, however, specify that you want to install the `enforcer` profile by including the `--profiles=` option and passing it the `enforcer` value
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                                              |        |                                      |            |                  |               |           |  observer:66s  
+  gke_cgacademy-project_us-central1_cluster-c | tester | 78ba1a81-a2d7-4ea2-a11c-dbee4e9776d1 |       141m | v1.24.9-gke.3200 |       7e170b0 |       65s |  observer:65s  
+```
+
+The rightmost column, labeled `ACTIVITY`, indicates when a cluster's profiles were last seen as "active" by the agent. If a cluster is enrolled under the `enforcer` profile, both `enforcer` and `observer` will be listed in the `ACTIVITY` column. If a cluster is enrolled under `observer`, though, only `observer` will appear. The previous sample output shows that `cluster-a` and `cluster-b` are enrolled under both the `enforcer` and `observer` profiles, while `cluster-c` is enrolled solely under `observer`. 
+
+You can also check the Chainguard Enforce Console to find what profiles your clusters are enrolled under. Navigate to the Console by going to [console.enforce.dev](https://console.enforce.dev) in your web browser. After signing in, there will be a table listing all of your clusters overseen by Chainguard Enforce.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md:80]: I thought "available to" or "enrolled in" may offer a more neutral tone here

```suggestion
You can also check the Chainguard Enforce Console to find what profiles your clusters are enrolled under. Navigate to the Console by going to [console.enforce.dev](https://console.enforce.dev) in your web browser. After signing in, there will be a table listing all of your clusters available to Chainguard Enforce.
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+![Screenshot of a portion of the Chainguard Enforce console, showing three rows representing three GKE clusters. The third column shows that the first and third clusters are enrolled in the enforcer and observer profiles, and the second cluster is only enrolled in the observer profile.](profiles_console.png)
+
+The **Profiles** column lists the profiles that each cluster is enrolled in.
+
+
+## Learn More
+
+The `observer` profile can be useful for testing out Chainguard Enforce and seeing how it works. You can also test out Enforce on a per-policy basis with `warn` mode. Check out our documentation on [disabling policy enforcement](../how-to-disable-policy-enforcement/) to learn more.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md:89]:
```suggestion
The `observer` profile can be useful for testing out Chainguard Enforce and understanding how it works. You can also test out Enforce on a per-policy basis with `warn` mode. Check out our documentation on [disabling policy enforcement](../how-to-disable-policy-enforcement/) to learn more.
```

**Review by ltagliaferri — COMMENTED** [review]
This is great! I left a few comments and copyedits. One structural question I had for you is whether you may want to lead with the default "Enforcer" profile. I guess the "Observer" profile is maybe more interesting to document but I was still expecting to read about the default first. Just curious what you thought.

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md (hunk 1 above)
Comment (SharpRake) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md:19]: Definitely with you on the "read-only" part, though I think the subj/verb agreement part is okay as it is. I did try changing it but I feel like this wording is as clear as I can make it.

At least how i read it, the broader permissions are what are doing the enabling here, so "X has broader permissions that enable you to...." scans. LMK if you still disagree and I'll try some more wordsmithing.

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md (hunk 5 above)
Comment (SharpRake) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md:89]: Yikes! My bad, good catch on this one

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md (hunk 1 above)
Comment (ltagliaferri) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md]: What about "The broader permissions of the `enforcer` profile enables you to benefit from all of Chainguard Enforce's features, including policy distribution and enforcement."

Comment (SharpRake) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md]: Works for me :+1:

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**